compiler/rustc_target/src/asm/mod.rs

```diff
@@ -1138,8 +1138,6 @@ impl InlineAsmClobberAbi {
                     f8, f9, f10, f11, f12, f13,
 
                     // v0-v19
-                    // FIXME: PPC32 SysV ABI does not mention vector registers processing.
-                    // https://refspecs.linuxfoundation.org/elf/elfspec_ppc.pdf
```

Comment:
Suggestion: can you replace this with the link in the PR description? Can't hurt to have some useful links here :3

Reply:
Added links (there are six links).

https://github.com/rust-lang/rust/blob/e2d50315efb9bcf5ad73bd4cb77f3d8f72a7d26a/compiler/rustc_target/src/asm/mod.rs#L1131-L1144

Reply:
Yes, that's very nice, thank you!